src/mlpack/tests/kde_test.cpp

```diff
@@ -921,17 +918,17 @@ BOOST_AUTO_TEST_CASE(NotTrained)
   KDE<>::Tree queryTree(query, oldFromNew);
 
   // Check results.
-  BOOST_REQUIRE_THROW(kde.Evaluate(query, estimations), std::runtime_error);
-  BOOST_REQUIRE_THROW(kde.Evaluate(&queryTree, oldFromNew, estimations),
+  REQUIRE_THROWS_AS(kde.Evaluate(query, estimations), std::runtime_error);
+  REQUIRE_THROWS_AS(kde.Evaluate(&queryTree, oldFromNew, estimations),
```

Comment:
I can't comment on the actual line, but can you fix the indentation in line 923.